admin_manual/configuration_server/server_tuning.rst

```diff
@@ -29,6 +41,20 @@ configure Nextcloud for MySQL or MariaDB. If your installation is already runnin
 SQLite then it is possible to convert to MySQL or MariaDB using the steps provided
 in :doc:`../configuration_database/db_conversion`.
 
+In smaller installations you might not want to set up a separate cache. However 
+you can still tune your database. The following example is suited for a database 
+smaller than 1GB. MySQL will consume up to 1GB of RAM for caching. Please make 
+sure that your system has sufficient free RAM after the change, so that it does 
+not start to use your swap partition, when it receives a burst of requests. In 
+the given example your ``/etc/mysql/conf.d/mysql.cnf`` might contain the 
+following lines. (beware that this is the block for mysqld not mysql)
+
+.. code:: ini
+
+  [mysqld]
+  innodb_buffer_pool_size=1073741824
```

Comment:
You could set this to 1G (https://dev.mysql.com/doc/refman/5.7/en/innodb-buffer-pool-resize.html).